fms/training/plugins.py

```diff
@@ -73,9 +68,16 @@ class InferenceValidator(TrainerPlugin):
     """
 
     def __init__(
-        self, prompt_tokens: List[str], tokenizer, device, steps=None, eos_token=None
+        self,
+        model: nn.Module,
```

Comment:
I like that we are now including the model here rather than in the function as not every function may use the model. (based on what I see below)